lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderService.java

```diff
@@ -52,56 +58,63 @@ public FusedLocationProviderService getService() {
     Log.d(TAG, "[onDestroy]");
   }
 
-  public Location getLastLocation() {
-    return serviceImpl.getLastLocation();
+  public Location getLastLocation(LostApiClient apiClient) {
```

Comment:
What do you think about renaming this param from `apiClient` to `client` in all the method signatures? It would be a bit more brief and also consistent with the [play services API](https://developers.google.com/android/reference/com/google/android/gms/location/FusedLocationProviderApi.html#getLastLocation(com.google.android.gms.common.api.GoogleApiClient)).